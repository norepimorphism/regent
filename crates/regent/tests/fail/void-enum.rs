fn main() {
    #[regent::bitwise]
    enum Void {}
}
