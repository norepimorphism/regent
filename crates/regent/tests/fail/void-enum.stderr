error: enum cannot have zero variants
 --> tests/fail/void-enum.rs:3:15
  |
3 |     enum Void {}
  |               ^^
